src/test/java/blackjack/domain/card/CardNumberTest.java

```diff
@@ -0,0 +1,20 @@
+package blackjack.domain.card;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+public class CardNumberTest {
+
+    @ParameterizedTest(name = "input : {0}")
+    @CsvSource(value = {"A:ACE", "2:TWO", "3:THREE", "4:FOUR", "5:FIVE", "6:SIX", "7:SEVEN", "8:EIGHT", "9:NINE", "K:KING", "Q:QUEEN", "J:JACK"}, delimiter = ':')
+    @DisplayName("of()는 카드넘버를 문자로 받아 값을 반환해 준다.")
+    void get_value_of_card_number_test(String number, String value) {
+        // given & when
+        CardNumber expected = CardNumber.valueOf(value);
+
+        // then
+        Assertions.assertThat(CardNumber.of(number)).isEqualTo(expected);
+    }
```

Comment:
위에서 언급드렸지만
테스트를 위한 코드를 만들어서 까지 parameterizedTest를 굳이 할 필요없을것 같습니다.
parameterizedTest는 테스트의 시간이 길어지게 하는 주요 원인이 될 수 잇을것 같아요
```suggestion
    @DisplayName("of()는 카드넘버를 문자로 받아 값을 반환해 준다.")
    @Test
    void get_value_of_card_number_test(String number, String value) {
        // given
        String value = "ACE";

        // when
        CardNumber expected = CardNumber.valueOf(value);

        // then
        Assertions.assertThat(CardNumber.ACE).isEqualTo(expected);
    }
```